components/ArticlesList.jsx

```diff
@@ -0,0 +1,21 @@
+import { getArticles } from "../api";
+import { useEffect, useState } from "react";
+import ArticleCard from "./ArticleCard";
+
+export default function ArticlesList() {
+    const [articles, setArticles] = useState([])
+
+    useEffect(() => {
+        getArticles().then(articles => {
+            setArticles(articles)
+        })
+    }, []);
+
```

Comment:
I would recommend putting some loading state in here to display a loading message whilst waiting for the articles to be fetched